tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -57,7 +58,7 @@ public abstract class MongoClientJsonDrivenTestRunnerBase
             "getnonce"
         };
 
-        private readonly string[] _expectedSharedColumns = { "_path", "database_name", "collection_name", "data", "tests", "minServerVersion" };
+        private readonly string[] _expectedSharedColumns = { "_path", "runOn", "database_name", "collection_name", "data", "tests" };
```

Comment:
Probably we can also add here `skipReason`, otherwise in the current version we will have an exception if the test contains `skipReason`

Reply:
Also, this property can be updated to make using the values from the base level more easier in inheritors. It's not really easy to make something like this:
        
        protected override string[] ExpectedSharedColumns
        {
            get
            {
                var list = new List<string>();
                list.AddRange(base.ExpectedSharedColumns);
                list.AddRange(
                    new[]
                    {
                       "field1", "field2"
                    });
                return list.ToArray();
            }
        }

Reply:
The second one probably is out of the scope of this ticket. Just thought out loud.

Reply:
None of the tests have `skipReason`. I propose we make no further changes for now.